cluster/gce/configure-vm.sh

```diff
@@ -762,7 +762,7 @@ users:
 clusters:
 - name: local
   cluster:
-    server: https://kubernetes-master
+    server: https://${1}:443
```

Comment:
isn't create-salt-kubelet-auth called from lots of places? need to update all the call sites or find a different way to pull this info